app/src/main/java/ru/suslovalex/androidacademyapp/data/entity/relation/MovieWithActors.kt

```diff
@@ -0,0 +1,16 @@
+package ru.suslovalex.androidacademyapp.data.entity.relation
+
+import androidx.room.Embedded
+import androidx.room.Relation
+import ru.suslovalex.androidacademyapp.data.entity.ActorEntity
+import ru.suslovalex.androidacademyapp.data.entity.MovieEntity
+
+data class MovieWithActors(
```

Comment:
@AleksandrAlex Do we really need this class. Looks like `MovieWithActorsAndGenres` is enough